druid-shell/src/platform/gtk/window.rs

```diff
@@ -854,7 +863,13 @@ impl WindowHandle {
 
     pub fn show_titlebar(&self, show_titlebar: bool) {
         if let Some(state) = self.state.upgrade() {
-            state.window.set_decorated(show_titlebar)
+            state.titlebar.set_visible(show_titlebar);
+        }
+    }
+
+    pub fn set_title(&self, title: impl Into<String>) {
+        if let Some(state) = self.state.upgrade() {
+            state.titlebar.set_title(Some(&(title.into())));
```

Comment:
Set the title on the window as well.
Maybe setting it on the window sets it on the HeaderBar widget as well ? I'll have to test this.

Reply:
Just to be sure, I decided to set it on both.